be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -317,7 +317,7 @@ Status ExchangeSinkOperator::Channel::_close_internal(RuntimeState* state, Fragm
 
 Status ExchangeSinkOperator::Channel::close(RuntimeState* state, FragmentContext* fragment_ctx) {
     auto status = _close_internal(state, fragment_ctx);
-    state->log_error(status.message());
+    state->log_error(status);
```

Comment:
It's better to comment in the code why use this function other than original one.

Reply:
There are enough comments in the definition of `RuntimeState`.
```cpp
    // Appends error to the _error_log if there is space
    bool log_error(std::string_view error);

    // If !status.ok(), appends the error to the _error_log
    void log_error(const Status& status);
```

Reply:
The lock information can be added.

Reply:
Updated.
```cpp
    state->log_error(status); // Lock only when status is not ok.
```